### ISSUE TITLE
[BUG]: Winning Post 7: Many black screens on hardware renderer, possible missing marker

### ISSUE THREAD

**Issue by HeroponRikiBestest** [PCSX2/pcsx2#4924]
### Describe the Bug

At many points in Winning Post 7, during various menus and during races, the screen will go completely black on the hardware renderer.

Additionally, from what I can tell, there should be a colored diamond marker over the player's horse during races, judging from footage. While it's possible that this only appears under certain conditions which I did not meet, this seems unlikely.
![image](https://user-images.githubusercontent.com/50224630/138264546-4c0dc07a-c7ea-4537-afad-930b82c18795.png)

### Reproduction Steps

Using the attatched memory card:
[Mcd001.ps2.zip](https://github.com/PCSX2/pcsx2/files/7388364/Mcd001.ps2.zip)
Mash circle until the game has been loaded and you're at a menu screen.
To get to a black screen quickly: while over the upper left most option, press circle twice, and it should take you to a menu that displays as a black screen on the hardware renderer.
To enter a race: select next, and once you've reached a different menu screen, select レース.

### Expected Behavior

I expected the game to display all menus and races on the hardware renderer, and I expected to see a marker over my player's horse during races.

### PCSX2 Revision

PCSX2 1.7.0-0 - compiled on Oct 18 2021 (daily ppa doesnt show git number)

### Operating System

Linux (64bit) - Specify Distro Below

### If Linux - Specify Distro

Linux Mint 20.2

### CPU

AMD Ryzen 7 PRO 4750U with Radeon Graphics

### GPU

integrated graphics, see above

### GS Window Screenshots

SW menu:
![gs_20211021055858](https://user-images.githubusercontent.com/50224630/138265029-05ccd70e-6fd9-4e47-86ca-2ed7a4b6f061.png)
HW menu:
![gs_20211021055955](https://user-images.githubusercontent.com/50224630/138265059-64ec5ee2-57ed-42e3-8e9a-f097bfebbfba.png)
SW race:
![gs_20211021061822](https://user-images.githubusercontent.com/50224630/138265088-2bdecde5-46d4-49f4-977a-e574a3e684b5.png)
HW race:
![gs_20211021055955](https://user-images.githubusercontent.com/50224630/138265059-64ec5ee2-57ed-42e3-8e9a-f097bfebbfba.png)

### Logs & Dumps

Menu:
[gs_20211021055858.gs.zip](https://github.com/PCSX2/pcsx2/files/7388412/gs_20211021055858.gs.zip)
Race:
[gs_20211021061822.gs.zip](https://github.com/PCSX2/pcsx2/files/7388420/gs_20211021061822.gs.zip)

**Comment by HeroponRikiBestest** [comment]
I forgot that the GI/G1 Jockey and Gallop Racer series share the same engine, so I need to look into which of those games also have the same issue.

**Comment by oumasanhasoushoku** [comment]
As you said, the same problem occurs in G1 Jockey 3 and later.
However, the Galloping Racer series does not have the same problem.

**Comment by HeroponRikiBestest** [comment]
Understandable. I'm sorry I haven't finished this yet. I was waiting on the x64 mipmapping issues to be fixed, but I think that might not be necessary.

**Comment by lightningterror** [comment]
Game needs Preload Frame Data to work, would be nice to give a list of all the affected games so we can add them to the gamedb in one bulk.

**Comment by oumasanhasoushoku** [comment]
As far as I have been able to verify.

- Winning Post 6 SLPM-65346
- G1 Jockey 3 2003 & Winning Post 6  PremiumPack SLPM-65385
- Winning Post 6 (Koei the Best) SLPM-66173
- Winning Post 6 (Koei Teiban Series) SLPM-66559
- Winning Post 6 2004 SLPM-65582
- Winning Post 6 2005 Nendoban SLPM-65894
- G1 Jockey 3 2005 Nendoban & Winning Post 6  2005 Nendoban SLPM-65893
- Winning Post 7 SLPM-66087
- Winning Post 7 (Koei the Best) SLPM-66811
- G1 Jockey 4 & Winning Post 7 TwinPack SLPM-66228
- Winning Post 7 Maximum 2006 SLPM-66342
- Winning Post 7 Maximum 2007 SLPM-66702
- G1 Jockey 4 2007 & Winning Post 7 Maximum 2007 PremiumPack SLPM-66904
- Winning Post 7 Maximum 2008 SLPM-66950
- G1 Jockey 4 2008 & Winning Post 7 Maximum 2008 PremiumPack SLPM-55073?
- Winning Post World SLPM-55150
- Winning Post World 2010 SLPM-55253

Click here for a list of the G1 Jockey Series.

- G1 Jockey 3 SLPM-62277 SLES-51357 SLUS-20690
- G1 Jockey 3 & Winning Post 5 Maximum 2002 PremiumPack SLPM-62279
- G1 Jockey 3 (Koei the Best) SLPM-62536
- G1 Jockey 3 (Koei Teiban Series) SLPM-62747
- Winning Post 5 Maximum 2003 SLPM-65298 (G1 Jockey 3 during)
- G1 Jockey 3 2003 SLPM-62392
- G1 Jockey 3 2003 & Winning Post 6  PremiumPack SLPM-62393
- G1 Jockey 3 2005 Nendoban SLPM-62591
- G1 Jockey 3 2005 Nendoban & Winning Post 6  2005 Nendoban SLPM-62590
- G1 Jockey 4 SLPM-66229 SLES-53917
- G1 Jockey 4 & Winning Post 7 TwinPack SLPM-66227
- G1 Jockey 4 (Koei the Best) SLPM-66812
- G1 Jockey 4 2006 SLPM-66538
- G1 Jockey 4 2007 SLPM-66888
- G1 Jockey 4 2007 & Winning Post 7 Maximum 2007 PremiumPack SLPM-66903
- G1 Jockey 4 2008 SLPM-55072
- G1 Jockey 4 2008 & Winning Post 7 Maximum 2008 PremiumPack SLPM-55073

**Comment by refractionpcsx2** [comment]
Is that a regression? or was that how it was back in 1.6?

**Comment by refractionpcsx2** [comment]
Can this be retested? maybe against #8290 ?

**Comment by oumasanhasoushoku** [comment]
![Winning Post 7_SLPM-66087_20230301153508](https://user-images.githubusercontent.com/62700805/222063191-71aff560-5ef5-42ad-83b2-052a608ddd41.png)

Seems normal except for some vertical lines appearing.

[Winning Post 7_SLPM-66087_20230301153508.zip](https://github.com/PCSX2/pcsx2/files/10857805/Winning.Post.7_SLPM-66087_20230301153508.zip)

PCSX2 1.7.4163

**Comment by refractionpcsx2** [comment]
That looks like you manually enabled round sprite full.

Anyway, that's an upscaling thing, which is out of scope of this issue, so I'll consider it done :)

**Comment by oumasanhasoushoku** [comment]
Since this issue has been resolved (although for some reason it was reopened)
I think it is safe to close.

**Comment by refractionpcsx2** [comment]
it was closed and reopened because back on March 1st it wasn't classed as completed, I did that by accident.

Anyways thanks for letting us know!
